07-Customizing-AzSK-for-your-Org/Extending AzSK Module/Readme.md

```diff
@@ -269,65 +272,64 @@ class SubscriptionCore: SVTBase
     ```
 
-2) Copy Storage.ps1 from module folder ->        <policyfolderpath>\Storage.ext.ps1
+2) Copy Storage.ps1 from module folder ->  <localPolicyfolderpath>\Storage.ext.ps1
 
-    a)  Clean up all but constructor, change to StorageExt, base-class to Storage, remove any class members.
+    a)  Remove all the methods except constructor, change class name to StorageExt, base-class to Storage, remove all class members.
 
-    b) Add method that will implement the new control
-    (see example file attached)
+    b) Add method that will implement the new control. Find sample script StorageExt.ps1 [here](../StorageExt.ps1)
 
-          
-3) Copy Storage.json from module folder -> <policyfolderpath>\Storage.ext.json
+
+3) Copy Storage.json from module folder to <localPolicyfolderpath>\Storage.ext.json
        
-    a) Remove all, except one existing control (to edit into new control JSON)
+    a) Remove all, except one existing control (to update it into new control JSON)
        
-    b) Edit the one control to reflect desired control-id, description, etc.
+    b) Edit the control to reflect desired controlId, description, etc.
        
     
     Few important things:
         
     i) ControlId and Id should not conflict with existing ones
             
-    ii) MethodName should correctly reflect the new method you wrote in Storage.ext.ps1 above
+    ii) MethodName should correctly reflect the new method written in Storage.ext.ps1 above
             
     iii) Do not change 'Storage' to 'StorageExt' inside the JSON (at the root node) 
 
-4) Update org policy... (Important: you don't need to edit anything else or manually tweak policy in the blob)!
+4) Update org policy using below command... (Important: Do not edit anything else or manually tweak policy in the blob)!
 
     ```Powershell    
     uop -SubscriptionId <subid> -OrgName <orgname> -DepartmentName <deptname> -PolicyFolderPath <policyfolderpath> 
     ```
 
 
-Important: Also note that the "ext" in the file-names above is in all lower-case!
+> Important: Also note that the "ext" in the file-names above is in all lower-case!
 
 ===============================================================
 
-Verifying that the added control works:
+5) Verifying that the added control works:
 
-1) Put your local AzSK into the target policy mode... (run "iwr" that is echoed by Step-4 above).
+1) Put the local AzSK into the target policy for that run "iwr" that is echoed by Step-4 above.
 
         
-2) Run grs for a storage accounts and verify that the control is getting scanned:
+2) Run grs for storage accounts and verify that the control is getting scanned:
         grs -s <subid> -rgns <resourcegroupname> -rtn Storage
 
-3) Run specific control-id, and across a couple of storage accounts in diff regions (the AzSKRG one will pass 'eastus2' check):
+3) Run specific controlId across a couple of storage accounts in diff regions (the storage present AzSKRG one will pass 'eastus2' check):
```

Comment:
1. Use "different" instead of "diff".
2. The sentence in the bracket is not clear: Does it mean "the storage present **in** AzSKRG"?

Reply:
Done